templates/crawler/{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,37 @@
+# {{cookiecutter.project_name}}
+
+Project skeleton generated by Crawlee (Beautifulsoup template).
+
+## Usage
+
+{% if cookiecutter.package_manager == 'poetry' %}
+To get started, ensure you have [Poetry](https://python-poetry.org/), a package and dependency management system, installed on your machine. We recommend installing it with the following command:
```

Comment:
could we wrap it to 120 line width?

Reply:
This is consistent how we write our docs - one paragraph per line. Is there a reason to do it differently here?

Reply:
no, sorry, I don't know why I wrote it :smile:, this is correct